bigquery/spec/bigquery_sample_spec.rb

```diff
@@ -276,6 +276,124 @@ def wait_until times: 5, delay: 1, &condition
       expect(loaded_data).to include({ name: "Bob",   value: 10 })
     end
 
+    example "import json data from GCS as new table" do
+      expect(@dataset.table "us_states").to be nil
+
+      capture do
+        import_table_from_gcs_json(
+          project_id:   @project_id,
+          dataset_id:   @dataset.dataset_id
+        )
+      end
+
+      expect(captured_output).to include(
+        "Importing data from Cloud Storage file: " +
+        "gs://cloud-samples-data/bigquery/us-states/us-states.json"
+      )
+      expect(captured_output).to match(
+        /Waiting for load job to complete: job/
```

Comment:
Here, you're using a regular expression without any expressions in it.

I would update this `.to include("Waiting...")` as you do right above

Reply:
(Here and in all of the tests :)